aat/src/aat/resources/features/F-081/F-081.feature

```diff
@@ -0,0 +1,43 @@
+@F-081
+Feature: CCD Definition Store Api :: GET /api/data/jurisdictions
```

Comment:
Please use the same name you suggested below, for consistency - Get Jurisdiction Details